test/parallel/test-fs-writev.js

```diff
@@ -0,0 +1,99 @@
+// Flags: --expose_externalize_string
+
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+const tmpdir = require('../common/tmpdir');
+
+tmpdir.refresh();
+
+const expected = 'ümlaut. Лорем 運務ホソモ指及 आपको करने विकास 紙読決多密所 أضف';
+
+const getFileName = (i) => path.join(tmpdir.path, `writev_${i}.txt`);
+
+/* eslint-disable no-undef */
+common.allowGlobals(externalizeString, isOneByteString, x);
+
+/**
+ * Testing with a array of buffers input
+ */
+
+// fs.writev with array of buffers with all parameters
+{
+  const filename = getFileName(1);
+  fs.open(filename, 'w', 0o644, common.mustCall((err, fd) => {
+    assert.ifError(err);
+
+    const buffer = Buffer.from(expected);
+    const bufferArr = [buffer, buffer];
+
+    const done = common.mustCall((err, written) => {
```

Comment:
We should be verifying the 3rd argument passed to the callback (`buffers`) for each test.